From c42de25c1785fdf5b5f6e3ec585d1fc045137e34 Mon Sep 17 00:00:00 2001
From: Matheus Guilhermino <matheus.machadoguilhermino@windriver.com>
Date: Sat, 28 May 2022 03:45:38 +0000
Subject: [PATCH] Adjust puppetlabs-translate version requirement

puppetlabs-translate v2.0.0 removes support for Debian 7 and bumps up
the minimum Puppet version (both of those are irrelevant here since we
are on Debian 11 and the Puppet version is still inside the range).
All other changes introduced from v2.0.0 to v2.2.0 are added support
and minor fixes.
Therefore, it should be safe to use v2.2.0 without a problem.

Signed-off-by: Matheus Guilhermino <matheus.machadoguilhermino@windriver.com>
---
 metadata.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/metadata.json b/metadata.json
index 1d6c4a9..f8972e1 100644
--- a/metadata.json
+++ b/metadata.json
@@ -14,7 +14,7 @@
     },
     {
       "name": "puppetlabs/translate",
-      "version_requirement": ">= 1.0.0 < 2.0.0"
+      "version_requirement": ">= 1.0.0 <= 2.2.0"
     }
   ],
   "data_provider": null,
-- 
2.30.2

